server/routes/confirmedOnReport/confirmedOnReport.test.ts

```diff
@@ -51,64 +51,7 @@ describe('GET /prisoner-placed-on-report', () => {
       .expect(res => {
         expect(res.text).toContain('Your report for John Smith has been submitted for review')
         expect(res.text).toContain('123')
-        expect(res.text).toContain('John Smith’s preferred language is')
```

Comment:
As per ts file comment - doesn't all this deleting mean we can get rid of the mocks?